### ISSUE TITLE
BN层

### ISSUE THREAD

**Issue by Sir-Death** [BIT-DA/DCAN#1]
你好，我想问一下在训练过程中为什么要freeze BN层的参数

**Comment by BinhuiXie** [comment]
Hiello @Sir-Death , it's a common practice to freeze BN while training models. This is done because you usually start from some ImageNet pre-trained weights which were trained with large batch size (like 256 or larger), while in domain adaptation the batch size per domain is much smaller. You don't want to ruin good pre-trained statistics so freezing BN is a good idea.

**Comment by Sir-Death** [comment]
您好，为什么我运行代码时，准确率只有百分之几

**Comment by BinhuiXie** [comment]
Hi @Sir-Death which task do you experiment? And also double check your data list, learning rate, image size, pytorch version, etc.

**Comment by Sir-Death** [comment]
office31,A->W，我使用的是原代码中的参数，有什么需要调整的吗

**Comment by Sir-Death** [comment]
你好，我很想知道论文中attention可视化部分是怎么做的，能分享一下代码吗，非常感谢

**Comment by BinhuiXie** [comment]
Sorry for replying late. We have updated the code about  [attention visulization](https://github.com/BIT-DA/DCAN/tree/master/AttentionVisualization).

**Comment by Sir-Death** [comment]
Thank you very much

**Comment by besaman** [comment]
Hi, 
I do also get a very low accuracy. Did you figure out the reason?
